Add pull request guides (#323)
exercism/docs: building/github/README.md

# Contributing via GitHub

Contributions to the Exercism code base are made via [GitHub][github].

As indicated by its name, GitHub uses [git][git] as its version control system.

> If you're unfamiliar with Git, the best online resource is [Pro Git][pro-git].

Reviewer: ```suggestion
If you're unfamiliar with Git, we recommend looking at is [Pro Git][pro-git].
```

Let's not make a judgement on what's best etc, as even if it's right, it'll go stale.
